Source/Filters/Photo/YPCoverImageView.swift

```diff
@@ -7,6 +7,7 @@
 
 import UIKit
 import Photos
+import Stevia
```

Comment:
nit: I don't think this is strictly needed as it's imported elsewhere. Line 51 already invokes Stevia in the `setup()`